src/test/java/revai/integration/ClientListener.java

```diff
@@ -0,0 +1,82 @@
+package revai.integration;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+import okhttp3.Response;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import revai.RevAiWebsocketListener;
+import revai.models.streaming.Hypothesis;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.CountDownLatch;
+
+public class ClientListener implements RevAiWebsocketListener {
+
+  private CountDownLatch connectedLatch = new CountDownLatch(1);
+  private CountDownLatch closeLatch = new CountDownLatch(1);
+  private CountDownLatch finalHypothesisLatch = new CountDownLatch(1);
+  private Gson gson = new Gson();
+  private JsonObject connectedMessage;
+  private List<Hypothesis> partialHypotheses = new ArrayList<>();
+  private List<Hypothesis> finalHypotheses = new ArrayList<>();
+  private static final Logger LOG = LoggerFactory.getLogger(ClientListener.class);
```

Comment:
do we need this logging in the released version of the libary.

My concern is (1) it adds another dependency, and (2) we don't document how to configure the logging system, so its unclear where the log messages would go. probably by default they would go to the console, which the users may not want.

it seems like it was useful for debugging, but i would take it out before release (along with the dependencies in the POM)